src/react-components/src/components/01-atoms/Forms/Label.tsx

```diff
@@ -0,0 +1,28 @@
+/* AT-50 Label */
+import * as React from "react";
+import bem from "../../../utils/bem";
```

Comment:
This isn't being used.

Reply:
This was probably for my benefit—I'll end up using it when I get to this component's styles.